src/components/ProjectCreation.js

```diff
@@ -53,65 +146,113 @@ export default function ProjectCreation() {
+                            id="skills"
                             type="text"
-                            placeholder="Enter Project Description"
-                            onChange={(e) => (setDescription(e.target.value))}
+                            placeholder="Enter Project Skills and separate with commas. Ex:( c++, javascript, react )"
+                            onChange={(e) => setSkills(e.target.value)}
                         />
                     </div>
                 </div>
                 <div className="form-group" style={{ flex: 1 }}>
-                    <label htmlFor="tags">Tags</label>
-                    <textarea className="form-control"
-                        style={{ height: '100%' }}
-                        id="tags"
-                        type="text"
-                        placeholder="Enter Project Tags and separate with commas. Ex:( Web Programming, Game Design )"
-                        onChange={(e) => (setTags(e.target.value))}
-                    />
-                </div>
-                <div className="form-group" style={{ flex: 1 }}>
-                    <label htmlFor="skills">Skills</label>
-                    <textarea className="form-control"
+                    <label htmlFor="description">Description</label>
+                    <textarea
+                        className="form-control"
+                        id="description"
                         style={{ height: '100%' }}
-                        id="skills"
                         type="text"
-                        placeholder="Enter Project Skills and separate with commas. Ex:( c++, javascript, react )"
-                        onChange={(e) => (setSkills(e.target.value))}
+                        placeholder="Enter Project Description"
+                        onChange={(e) => setDescription(e.target.value)}
                     />
                 </div>
-                <div className="form-group" style={{ flex: 1 }}>
-                    <label htmlFor="links">Links</label>
-                    <textarea className="form-control"
-                        style={{ height: '100%' }}
-                        id="links"
-                        type="text"
-                        placeholder="Enter Project Links related to the project if needed"
-                        onChange={(e) => (setLinks(e.target.value))}
-                    />
+                <div className="tags-links-container" style={{ flex: 1 }}>
+                    <div className="form-group" style={{ height: '100%' }}>
+                        <label htmlFor="tags">Tags</label>
+                        <textarea
+                            className="form-control"
+                            style={{ height: '100%' }}
+                            id="tags"
+                            type="text"
+                            placeholder="Enter Project Tags and separate with commas. Ex:( Web Programming, Game Design )"
```

Comment:
"Enter project tags and separate with commas (e.g. web development, game design, crypto)"